.github/workflows/build.yaml

```diff
@@ -19,6 +19,10 @@ name: build
 
 on:
   workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
log4net main branch is called "master" currently.

Reply:
Corrected